### PR TITLE
Html forms (Noah Khayat)

### PR DESCRIPTION
**Pull request by NaeemK1188**
Screenshot (52)

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      textarea {
+        width: 250px;
+        height: 150px;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label>Email</label>
```
Comment (shawnkost) [inline html-forms/index.html]: The label needs a `for` that matches the `id` of the `input` you want to associate it with

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      <br />
+      <input type="email" name="email" required />
+      <br />
+      <label>Why is Coding Cool?</label>
```
Comment (shawnkost) [inline html-forms/index.html:22]: The label needs a `for` that matches the `id` of the `input` you want to associate it with

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          type="radio"
+          name="interest"
+          value="JavaScript" />JavaScript</label
+      >
+      <p>Would you like to sign up for more information?</p>
+      <label
+        ><input type="checkbox" name="check" value="Signup" />Sign me up!</label
+      >
+      <br />
+      <label>Comments</label>
```
Comment (shawnkost) [inline html-forms/index.html:58]: The label needs a `for` that matches the `id` of the `input` you want to associate it with

**Review by shawnkost — CHANGES_REQUESTED** [review]
Misisng some `for` & `id` attributes to connect your `label`s and `input`s

**Comment by NaeemK1188** [comment]
adding form and id attributes to the missing parts

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**